### PR TITLE
Add water trails to ships

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/ra2/issues/336.

The palette is wrong and none of the currently setup ones seems to fit.

### CODE REVIEW

**Comment by GraionDilach** [comment]
Make it 50% transparent. Translucent=yes was basically a rough fade-out translucent effect (25%-50%-75% steppings IIRC), so indeed there isn't an 1:1 solution.

**Comment by Mailaender** [comment]
Do you happen to know the water palette? I need to use something as base to make translucent.

File: mods/ra2/rules/soviet-naval.yaml [hunk 1]
```diff
@@ -64,6 +64,7 @@ sub:
 	Mobile:
 		TurnSpeed: 4
 		Speed: 71
+	-LeavesTrails:
```
Comment (Phrohdoh) [inline mods/ra2/rules/soviet-naval.yaml]: Should this be conditional based on whether or not the sub is submerged?

Comment (GraionDilach) [reply mods/ra2/rules/soviet-naval.yaml]: In RA2, leaving trails had nothing to do with cloaking, but to Underwater=yes, which disabled trails altogether if present (along with some other effects like sub targeting for NavalTargeting modes).

**Comment by Mailaender** [comment]
> Make it 50% transparent. Translucent=yes was basically a rough fade-out translucent effect (25%-50%-75% steppings IIRC), so indeed there isn't an 1:1 solution.

I just read about https://github.com/OpenRA/OpenRA/issues/10727, which is what you want me to fake here.

**Comment by GraionDilach** [comment]
And I just noticed that you were doubted on the actual palette - yes, effect is the correct one. Which is why I ignored the question.

**Comment by Mailaender** [comment]
Alright. Thanks.

I won't add the fade out effect, yet. At least not in this pull request.

**Comment by Phrohdoh** [comment]
Why not if it is part of this feature?

**Comment by Mailaender** [comment]
Personal time constraints and this probably being more of an upstream engine feature rather than exclusive mod code.

**Comment by GraionDilach** [comment]
I'm still voting for effect50 (akin to how it's set up in Dune2000) as a compromise.

**Comment by Mailaender** [comment]
> I'm still voting for effect50 (akin to how it's set up in Dune2000) as a compromise.

Updated and called it `effect50alpha`.

**Comment by DylanC** [comment]
@Mailaender  - Can this be merged in?

**Comment by Mailaender** [comment]
@abcdefg30 is the current maintainer of the ra2 mod. He and the reviewers decide.

File: mods/ra2/sequences/misc.yaml [hunk 2]
```diff
@@ -277,6 +277,14 @@ moveflsh:
 		Length: *
 		Tick: 30
 
+wake:
+	idle: wake2
```
Comment (MustaphaTR) [inline mods/ra2/sequences/misc.yaml]: This is wrong. Correct file is the wake1.shp under conquer.mix. There are wake1.shps under other mixes too tho so just writing wake1 doesn't work. This needs a setup like TS cameos use.

Comment (Mailaender) [reply mods/ra2/sequences/misc.yaml]: Corrected that.

**Pull request merged**